docs/design/coreclr/botr/clr-abi.md

```diff
@@ -666,12 +692,12 @@ EnC is not supported for generic methods and methods on generic types.
 # System V x86_64 support
 
 This section relates mostly to calling conventions on System V systems (such as Ubuntu Linux and Mac OS X).
-The general rules outlined in the [System V x86_64 ABI](https://software.intel.com/sites/default/files/article/402129/mpx-linux64-abi.pdf) are followed with a few exceptions, described below:
+The general rules outlined in the System V x86_64 ABI documentation are followed with a few exceptions, described below:
 
-1. The hidden argument for by-value passed structs is always after the "this" parameter (if there is one). This is a difference with the System V ABI and affects only the internal JIT calling conventions. For PInvoke calls the hidden argument is always the first parameter since there is no "this" parameter in this case.
+1. The hidden argument for by-value passed structs is always after the `this` parameter (if there is one). This is a difference with the System V ABI and affects only the internal JIT calling conventions. For PInvoke calls the hidden argument is always the first parameter since there is no `this` parameter in this case.
```

Comment:
> For PInvoke calls the hidden argument is always the first parameter since there is no `this` parameter in this case.

It should not be the case for CallConvMemberFunction https://github.com/dotnet/runtime/pull/47828/

Reply:
I only changed the formatting of `this` -- probably shouldn't have done that...

Anyway, I added, " (except for the `CallConvMemberFunction` case)."